package.json

```diff
@@ -131,15 +131,14 @@
     "chromatic": "^6.17.1",
     "codecov": "^3.8.3",
     "copyfiles": "^2.4.1",
-    "eslint": "^8.34.0",
+    "eslint": "^8.35.0",
     "eslint-config-prettier": "^8.6.0",
     "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-storybook": "^0.6.11",
     "eslint-plugin-testing-library": "^5.10.2",
     "husky": "^8.0.3",
     "identity-obj-proxy": "^3.0.0",
-    "intersection-observer": "^0.12.2",
```

Comment:
Headless now uses ResizeObserver instead of intersection-observer. Could have used similar ResizeObserver polyfill but they were ancient and just decided to assign it in the test file